corehq/apps/tour/models.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+from django.contrib.auth.models import User
+from django.db import models
+import json_field
+
+
+class GuidedTours(models.Model):
+    user = models.ForeignKey(
+        User,
+        on_delete=models.PROTECT,
+        unique=True
+    )
+    seen_tours = json_field.JSONField(
+        default={},
+    )
+
+    @classmethod
+    def has_seen_tour(cls, user, tour_slug):
+        guided_tour, _ = GuidedTours.objects.get_or_create(user=user)
+        return tour_slug not in guided_tour.seen_tours
```

Comment:
but before I do that, what's your take on how that will scale assuming that for every `User` x `GuidedTour`, there will be a new row in this table? I think we touched on this earlier, and I picked the solution that seemed like it would take up fewer rows. I don't think we'll be writing / reading as often to the DB for that part to actually be a concern. But I'm less familiar with the inner workings of how the data will actually be handled, and am defaulting to your recommendation here.